src/fu-util.c

```diff
@@ -2524,7 +2529,10 @@ main (int argc, char *argv[])
 		{ "no-history", '\0', 0, G_OPTION_ARG_NONE, &no_history,
 			/* TRANSLATORS: command line option */
 			_("Do not write to the history database"), NULL },
-		{ "show-all-devices", '\0', 0, G_OPTION_ARG_NONE, &priv->show_all_devices,
+		{ "show-all", '\0', 0, G_OPTION_ARG_NONE, &priv->show_all,
+			/* TRANSLATORS: command line option */
+			_("Show all results"), NULL },
+		{ "show-all-devices", '\0', G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_NONE, &priv->show_all,
```

Comment:
this line should be dropped with a rename of the option

Reply:
Ohh, I don't think there's any way we can drop it; `G_OPTION_FLAG_HIDDEN` means it's not in --help any more, but there are hundreds of places all over the internet that specify running `fwupdmgr --show-all-devices` -- it's essentially part of our CLI "ABI" now.

Reply:
Ohhh I see.  In that case, I wonder if we actually should have kept `--plugin-whitelist` to alias to `--plugins` like that too.

Reply:
I think so too; I've added it as a follow up to this branch.